engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-ostree-repositories.html

```diff
@@ -25,90 +23,67 @@ <h3 translate>
 </nav>
 
 <div data-extend-template="layouts/partials/table.html">
-  <div data-block="search">
-    <div class="col-sm-6">
-      <form class="form-inline">
-        <div class="form-group">
-            <select class="form-control" ng-model="product" ng-options="product.name for (id, product) in products"></select>
-            <input type="text"
-                   class="form-control"
-                   placeholder="{{ 'Filter' | translate }}"
-                   ng-model="repositorySearch"/>
-
-        </div>
-      </form>
-    </div>
-  </div>
-
   <div data-block="list-actions">
-    <button class="btn btn-default"
+    <button class="btn btn-primary"
             ng-disabled="table.numSelected === 0"
             ng-show="isState('content-view.repositories.ostree.list') && permitted('edit_content_views', contentView)"
             ng-click="removeRepositories(contentView)">
       <span translate>Remove Repositories</span>
     </button>
-
-    <div ng-show="repositoriesTable.rows.length === 0 && !repositoriesTable.working">
-      <p bst-alert="info" ng-show="isState('content-view.repositories.ostree.list')">
-        <span translate>
-          There are currently no repositories associated with this Content View, you can add some by clicking on the "Add" tab above.
-        </span>
-      </p>
-      <p bst-alert="info" ng-show="isState('content-view.repositories.ostree.available')">
-        <span translate>There are currently no repositories to add to this Content View,</span>
-        <a ui-sref="products.index" translate>add some repositories.</a>
-      </p>
-    </div>
+    <button class="btn btn-primary"
+            ng-disabled="table.numSelected === 0"
+            ng-show="isState('content-view.repositories.ostree.available') && permitted('edit_content_views', contentView)"
+            ng-click="addRepositories(contentView)">
+      <span translate>Add Repositories</span>
+    </button>
   </div>
 
   <span data-block="no-rows-message">
     <span ng-show="isState('content-view.repositories.ostree.list')" translate>
-      There are currently no OSTree Repositories associated with this Content View, you can add some by clicking on the "Add" tab above.
+      There are currently no Repositories associated with this Content View, you can add some by clicking on the "Add" tab above.
```

Comment:
That doesn't really answer the question.  Aren't OSTree Repositories distinct from yum repositories?  To me having this extra descriptor was useful.